extensions/interactions/MultipleChoiceInput/static/multiple_choice_input.css

```diff
@@ -45,7 +40,26 @@
   width: 8px;
 }
 
-.multiple-choice-option:hover .multiple-choice-outer-radio-button,
 .multiple-choice-option:focus .multiple-choice-outer-radio-button {
   border-color: #115FD4;
 }
+
+/* Hover pseudo-class should be used in desktops only. */
```

Comment:
Why?

Reply:
Hover is not relevant in mobile devices, I think. Also, we are going to use a button for multiple choice input now.

Reply:
ditto

Reply:
Done.